index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-session">
+      <h2>Login</h2>
+      <h3>Enter your credentials</h3>
+      <a href="./register.html" id="register-html">Register here</a>
+      <form onsubmit="return false;" class="login-form">
+        <select id="select" required>
```

Comment:
Avoid vague names for your identifiers like "select". It's too vague and makes reading your code very difficult. Give it a more descriptive name, such as "account-number".

Secondly, you should use the form component wrapper ("form-group", "form-control", etc) we used on the register page in order to achieve site-wide layout consistency.